notebooks/14-combine-data.md

```diff
@@ -173,7 +175,16 @@ xr.open_dataset("moisture_index_stacked.nc", engine="netcdf4")
 Storing to zarr files works on the `xarray.DataSet` level:
 
 ```{code-cell} ipython3
-moisture_index_lazy.to_zarr("moisture_index_stacked.zarr")
+moisture_index_lazy.moisture_index
+```
+
+```{code-cell} ipython3
+moisture_index_lazy
+```
+
+```{code-cell} ipython3
+moisture_index_lazy.to_zarr("moisture_index_stacked.zarr", 
+                            encoding={"moisture_index": {"dtype": "float32"}})  # explicitly defines the data type to store
```

Comment:
It's bizarre we need this change. I can't make sense of the warning otherwise:

```
SerializationWarning: saving variable None with floating point data as an integer dtype without any _FillValue to use for NaNs
```

Reply:
aha, ok that is even better.